scribble-test/tests/scribble/docs/manual.scrbl

```diff
@@ -99,6 +99,14 @@ A function, again, not a link target, documented to return @racket[10] using a d
 
 @defstruct[#:link-target? #f pt ([x real?] [y real?]) #:mutable]{A mutable structure type with extra name, again.}
 
+@defstruct*[#:link-target? #f counter ([name symbol?] [(count) exact-nonnegative-integer?])]{A struct with an (empty) list of field options.}
+
+@defstruct*[counter ([name symbol?] [(count #:mutable) exact-nonnegative-integer?])]{Another struct with a mutable field.}
+
+@defstruct*[#:link-target? #f counter ([name symbol?] [(count #:auto-value) exact-nonnegative-integer?])]{A struct with an automatic field.}
```

Comment:
It's called `#:auto`, not `#:auto-value`, FWIW.

Reply:
Fixed. Thanks!

For the `field-option` checks, it seems like most similar checks in Scribble are syntax errors. I don't know how to do it so I'll leave that out.